### ISSUE TITLE
Google Chrome resizing/re-positioning weirdly

### ISSUE THREAD

**Issue by GRardB** [kasper/phoenix#279]
- Version: 2.6.7 (98)
- macOS: 11.4 (20F71) (on M1 MacBook Air)

Very recently, I encountered a bug(?) which only occurs with Google Chrome. In the attached video, I am "centering" and maximizing two windows: one from Finder, and one from Chrome.

https://user-images.githubusercontent.com/748180/124414030-e162b800-dd1f-11eb-91bf-61ea427e9b57.mov

This is the code for these behaviors:

```js
Key.on('m', ['control', 'option', 'command'], function() {
  const w = Window.focused()

  w.maximize()
})

Key.on('c', ['control', 'option', 'command'], function() {
  const w = Window.focused()

  w.setTopLeft({
    x: getFullWidth() * 0.125,
    y: getFullHeight() * 0.125,
  })

  w.setSize({
    width: getFullWidth() * 0.75,
    height: getFullHeight() * 0.75,
  })
})
```

For the Finder window, everything works as expected. My keyboard shortcut sequence is:

1. cmd + optional + command + c
1. cmd + optional + command + m
1. cmd + optional + command + c

With Chrome, however, only one action happens for each shortcut attempt. The sequence shown in the video is:

1. cmd + optional + command + c
1. cmd + optional + command + c
1. cmd + optional + command + c
1. cmd + optional + command + m
1. cmd + optional + command + m
1. cmd + optional + command + m
1. cmd + optional + command + c
1. cmd + optional + command + c

Basically, each time the function runs, it only performs one step in the process towards the desired final state, skipping any steps whose end states are already reached. Also noteworthy is that the Chrome window resizes and moves smoothly, which didn't happen until recently.

Electron apps don't suffer from the same issue, nor do Firefox or Safari. Lastly, this happens with all of my configured actions, not just these two.

I think this is likely an issue with Chrome and not Phoenix, but my wife doesn't experience this with [Rectangle](https://github.com/rxhanson/Rectangle) on the same versions of Chrome and macOS (AFAIK, at least). I wonder if there is a workaround for this, either on my end or on Phoenix's?

**Comment by GRardB** [comment]
I actually think I've fixed the issue. I can't say for sure, since I was doing a lot of things—all of which were unrelated to Phoenix—but my guess is that restarting my computer is what did it.

**Comment by goerwin** [comment]
I also started seeing this issue, after quitting Chrome and open it again fixed the issue, but still odd behavior.

**Comment by kasper** [comment]
@goerwin Sorry to hear, out of interest, does Chrome report the resize was successful through the API in these cases?

**Comment by goerwin** [comment]
Haven't been able to reproduce it again, but I also saw the same behavior for iTerm. But after closing/reopening them the issue went away. I haven't use the API for other than resize/move windows, but I can report back with that info if it happens again

**Comment by kasper** [comment]
@goerwin Thanks! Usually these should be just caused by the app not following the request or sizing constraints in the app not allowing a particular size, but some apps seem to be more prone to errors than others.
